cloudkittydashboard/dashboards/admin/summary/tables.py

```diff
@@ -37,9 +37,40 @@ class Meta(object):
 
 
 class TenantSummaryTable(tables.DataTable):
-    res_type = tables.Column('res_type', verbose_name=_("Res Type"))
+
+    def convert_string(word):
+        #  Converts string to readable format
+        return word.replace(
+            '_', ' ').title().replace(' Id', ' ID').replace('Id', 'ID')
+
+    groupby_list = ['type', 'id', 'user_id']
+
+    # Dynamically create columns based on groupby_list
+    for field in groupby_list:
+        locals()[field] = tables.Column(
+            field, verbose_name=_(convert_string(field)))
+
     rate = tables.Column('rate', verbose_name=_("Rate"))
 
+    def __init__(self, request, data=None, needs_form_wrapper=None, **kwargs):
+        super().__init__(request, data, needs_form_wrapper, **kwargs)
+
+        #  Hide columns based on checkbox selection
+        for field in self.groupby_list:
+            if request.GET.get(field) != 'true':
+                self.columns[field].classes = ['hidden']
+
     class Meta(object):
         name = "tenant_summary"
-        verbose_name = _("Project Summary")
+        verbose_name = _("Tenant Summary")
```

Comment:
We don't want to rename this
```suggestion
        verbose_name = _("Project Summary")
```

Reply:
This is the change I was talking about yeysterday (where I hadn't seen that someone committed a change and I was copying in all my changes from my dev branch), I  thought it had all changed successfully? Weird. This can be changed back of course, as this wasn't my intention!